R/downloadPR.R

```diff
@@ -10,6 +10,9 @@
 #' @param pr List or numeric list. Classically the returned object from
 #' \code{\link{getPR}}.
 #' @param year Numeric or list (i.e.: c(2000, 2005))
+#' list does not work anymore!!!
```

Comment:
Package is documented using roxygen2, so you need to use the rocklets 
everything behind `#'` is part of the inline documentation, so it needs to be indexed by rocklets (e.g. `@param` for parameters).
And before submitting a new pull request, it would be good to re-generate documentation by running the `roxygenize()` function. There might also be a button or even a keyboard shortcut to do that in RStudio